lib/user.rb

```diff
@@ -0,0 +1,36 @@
+require_relative "recipient"
+require "dotenv"
+Dotenv.load
+
+module SlackCLI
+  class User < Recipient
+    attr_reader :real_name
+
+    def initialize(slack_id:, name:, real_name:)
+      super(slack_id: slack_id, name: name)
+
+      @real_name = real_name
+    end
+
+    def self.list
+      url = "https://slack.com/api/users.list"
+      data = {
+        "token": ENV["SLACK_API_TOKEN"],
+      }
```

Comment:
Most of this data is going to be common to all subclasses of `Recipient`, the only thing that's changing is the end of the URL (`users.list`). Could you move some or all of it to `Recipient.get`?